schemas/preferences.sublime-settings.json

```
@@ -458,7 +499,7 @@
           "default": true
         },
         "indent_guide_options": {
-          "markdownDescription": "Controls how the indent guides are drawn, valid options are \"draw_normal\" and \"draw_active\". `draw_active` will draw the indent guides containing the caret in a different color.",
+          "markdownDescription": "Controls how the indent guides are drawn, valid options are:\n\n- `\"draw_normal\"`: Draw all indent guides in the same color.\n- `\"draw_active\"`: Draw the indent guides containing the caret in a different color.\n- `\"draw_active_single\"`: Draw only the active indent guide containing the caret.\n- `\"solid\"`: Draw solid vertical lines instead of dotted.\n",
```

Removed trailing newline (probably makes no difference visually).
```suggestion
          "markdownDescription": "Controls how the indent guides are drawn, valid options are:\n\n- `\"draw_normal\"`: Draw all indent guides in the same color.\n- `\"draw_active\"`: Draw the indent guides containing the caret in a different color.\n- `\"draw_active_single\"`: Draw only the active indent guide containing the caret.\n- `\"solid\"`: Draw solid vertical lines instead of dotted.",
```